roles/etcd/tasks/configure.yml

```diff
@@ -1,6 +1,7 @@
 ---
 - name: Configure | Check if etcd cluster is healthy
-  shell: "{{ bin_dir }}/etcdctl --endpoints={{ etcd_access_addresses }} cluster-health | grep -q 'cluster is healthy'"
+  become: yes
+  shell: "{{ bin_dir }}/etcdctl --endpoints={{ etcd_access_addresses }} --cert-file {{ etcd_cert_dir }}/ca.pem --key-file {{ etcd_cert_dir }}/ca-key.pem cluster-health | grep -q 'cluster is healthy'"
```

Comment:
Using the CA certificate for authorization in etcd is wrong. 
You did not specify a CA certificate that allows you to verify the server.

All required certificates are already specified in environment variables.

I'm sure your changes are unwanted

Reply:
So are you saying that I need to use something like this?

```yaml
  environment:
    ETCDCTL_API: 3
    ETCDCTL_CERT: "{{ etcd_cert_dir }}/admin-{{ inventory_hostname }}.pem"
    ETCDCTL_KEY: "{{ etcd_cert_dir }}/admin-{{ inventory_hostname }}-key.pem"
    ETCDCTL_CACERT: "{{ etcd_cert_dir }}/ca.pem"
```

Reply:
I don't understand you. I can’t say exactly what you need to use.
I just want to indicate that in this task all the necessary certificates are already specified in the environment variables, and nothing needs to be changed

and also I want to clarify that the command `cluster-health` exists only in etcd APIv2